# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1494347679 14400
#      Tue May 09 12:34:39 2017 -0400
# Node ID bce43f1a3cced131abbd8bd76f567b5305b864e3
# Parent  3530b956d707c393c44aee9fa0f18d15d579bdb9
* __eigs__.cc: Avoid deprecated function warnings.

diff --git a/libinterp/dldfcn/__eigs__.cc b/libinterp/dldfcn/__eigs__.cc
--- a/libinterp/dldfcn/__eigs__.cc
+++ b/libinterp/dldfcn/__eigs__.cc
@@ -60,17 +60,17 @@ eigs_func (const ColumnVector& x, int& e
   args(0) = x;
 
   if (eigs_fcn)
     {
       octave_value_list tmp;
 
       try
         {
-          tmp = feval (eigs_fcn, args, 1);
+          tmp = octave::feval (eigs_fcn, args, 1);
         }
       catch (octave::execution_exception& e)
         {
           err_user_supplied_eval (e, "eigs");
         }
 
       if (tmp.length () && tmp(0).is_defined ())
         {
@@ -100,17 +100,17 @@ eigs_complex_func (const ComplexColumnVe
   args(0) = x;
 
   if (eigs_fcn)
     {
       octave_value_list tmp;
 
       try
         {
-          tmp = feval (eigs_fcn, args, 1);
+          tmp = octave::feval (eigs_fcn, args, 1);
         }
       catch (octave::execution_exception& e)
         {
           err_user_supplied_eval (e, "eigs");
         }
 
       if (tmp.length () && tmp(0).is_defined ())
         {
